tests/org.eclipse.xtend.core.tests/src/org/eclipse/xtend/core/tests/validation/ValidationBug427432Test.xtend

```diff
@@ -41,7 +41,7 @@ class ValidationBug427432Test extends AbstractXtendTestCase {
 				}
 			
 			}
-		'''.parse.assertError(XbasePackage.Literals.XFEATURE_CALL, IssueCodes.DUPLICATE_CASE)
+		'''.parse.assertNoErrors
```

Comment:
Maybe I just don't get it. We still have dead code in the test model but now we assert that we don't find it?